staging/src/k8s.io/apiextensions-apiserver/examples/client-go/pkg/client/informers/externalversions/internalinterfaces/factory_interfaces.go

```diff
@@ -35,4 +36,5 @@ type SharedInformerFactory interface {
 	InformerFor(obj runtime.Object, newFunc NewInformerFunc) cache.SharedIndexInformer
 }
 
+// TweakListOptionsFunc is a function that applies a transform on v1.ListOptions.
```

Comment:
Gah I wrote my comment late at night. "is a function that applies a tranformation to a"? "is a function that transforms a"?